.classpath

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
[medium] I wouldn't commit this file since its creation should be taken care by maven.